### ISSUE TITLE
Corgi Engine

### ISSUE THREAD

**Issue by reunono** [PeopleAtplay/TechnoTrigger#1]
Hi, 

Could you please remove the Corgi Engine files included in this project from Github ?
Thanks,
Renaud

**Comment by PeopleAtplay** [comment]
Yes totally ,I am a poor college student and was just looking at your
project.You do  great work! keep it up.
